index.html

```diff
@@ -7,6 +7,78 @@
VQ56mHBrZfKsuDEv8nx/gwaJOtqhrgLvEbElTCBVJahzhhaNWV518OCjcCRheDpBgwmpLLrTJcy1BbHJssmpSEf0SCprO0ivbodgJOlgPMuU+9/fkADHgxmn0k07/3y+nDOBEbThqzD81DEUQy/KxyJOj5LeKYjoFGXRNbfAAkKGGnCCSJ128ZziV7c8AOawi8d0yBhp6HhKXK3iZnsGQxhsXwcweDKljcaJGzC4FjRtcb00qUh5APmzIVx6vtvmeVhGvt5no5p4DUEob82go3tjxSc3I/T3WCEnj6mP8HEs09NYOCk/jB3th2zmuVYvjOOTNyBK97OdLEsh4PFV42nfJZUNFipaLBS0WClosFKRYMVbhoqZsrvO8asCQKfOQxkcYQRTJvxP4oof4Rw08jlRkPcT1GBS6hIBOcmZMwYVdu6Yp/ktF2v8NIwR4YkyfupcwdXEsAnjFQwMKuOpKOQyHULL42QrlCzdyMlR6qzNEhsa8fRsmsWXho+zZbdT8oc01DTz0rs+BThfauAZ2D6r7R7LcAxjYNzj2fojjOCDk/geW/DpcJJAxehdXBOb2sqj2k4cR7TAV2UtzvNdjHeAhCcuN1udtrpfpClOlM4Ybo94BbwbfFP8nlY4aRBQQASuVPuH9FQm0sbY5DuSLNlRVY0m65XUZuareO+YstbGxuKtgIH5OUIeJnNpQ47uj39x70zHw0Tl3C5gleX5FKxj2m0y/U5oYbRWAVTgLD7MRUZWhfUXRKlsgMOMZwNdAzFRxg44YrTN/Y/ydp4Ez4aIQa9VJzA374w4ZhGZ7taqb2Uvem0gRUcQnXrALHbLjp1XLIRlAtbRH1UTEcIB2ef7bY/1Y8jFJ8ufDSmtoi5A5m+mwt/j4Zb4PyOia8VAHpIoxtiZgtoB75qIceK42I2E07DyWEaoIztowjCpwsXDaFRF5cBjRiIIjV579HYSmTTJASkUU4Rdspw9YhdEJeDwpXBEV2SR/+0a+GiMdbo/GRpTGng430aQZhlWS7uaXT3oQWfqAZNwtjKVBHlNpycpQ1JuvgFOBcJ7xsnpDku2cRF0iM88h4NYdqQ6UqcMzRiouoSE6Dt0AAtruqBz/fe2vI5wkPDxAZShlNEsW7jU3yHRmTY4m9oBNoxjXJJlKLoX4eGY4uM0DfyvEOjrUPfkY/HqXxeN7a5P6XQ2JEbbuWfehw8NNr4qA0ZxAAtMXC593kaqERdjAEEsnSOBgYhtf1QF9+2wBEh4hEOGlEXEIxdKhn0Lvh0z9MQtG3gKHqHBi6UpfFsYpokAJXTCuxLzJQjiniJcNDYv20DBRo7pp+9oxsYvJ8KphkqZ1sKcVG35GmaFpgJiq6o7flp7Nnev3XNOWhg/sHec051mpF2PNuztRs+utqNRkOUztOA1gFOuqzrCiINPOxNFF0x/rVrfjkNYQmDrr0mR9AlLkM6E1gnOBOo0JnApk5TQ835UpEMyTAkGJYBDUVTygSwtq2UFc4VG9RKkqg2BHNbq0sw+LGVfztxdjkNp9NuM2PMotNupiRsw1HYy+keUeNOu4PE1LTjeQ18vWIBDriJZ9HIalyeBxINR3DCaFpODYzbeHoz/scT+pfTUFFOdncH3z6256hCFAWBaZKDw2wZZhAFb0u2hCAS/vmETxVPYaWiwUpFg5WKBisVDVYqGqxUNFipaLDCNdtjvrlHJrqW6mGEzHx7BW1wNPoS9sutgoBg/qBKIoG9grA50riIpXTtyhuo7LfuNmT3EeM7njWPU3+4v9gdQlmFqRa0mhEdv4T+7n01RbFf6axmppmR8X4pX+E7wxA+HRy9R1OGZp4xW1kauyQfkgjKjDomUYvdar4ipDG57CPWUvPMb/gqPBa1DKdGOaVBShr401TVzbcPMMsxxGrSSKvpC8JQzdLy4boFKNjQVYlgpiGo0BALhLOgVJJGO02jNOCSoiAhlBw1QxJ0nK06DN0AVccUSr3h8ua5aGA9najI1Sj2KY1A8IM8CLPh1HWLYVgMy8Qet4CbOIWj5sU46+TZFKpmDn0A5+CQbTgsQjd0QnM4jN00NoeFEKZFCpi2kVhKA3gPh3T+xy1SkvmOmhYwaI4LF+5YxGEW5nA8LXgmiLhayjBMc9OP0jAPXUyqn0K9AAmGCszCFVw/8AMSQhtyplHgo1a48D8gMzMnHWPjopNbQzg/GztuHkZAMEizKA/TqCBZbKIuBSUNEkfpOIwBYZbFAvAM06Bwh6of+hE8BlCe3PXhxj6HdnDpRmCmkVrkfjpUaUvxBRMBZBn8UJzRytVhQHUbHmVYpNMgK4bq0BSGaAaGOc6V4QB+GAGIzAFsAfAleZznWWb6YBtAtcxc2NEAWjlms2cu8I/SrABlyMgwHGLDDNMsC4ZZkRb/zXpYbClpqILpNAsziku7ATQEpCHAl26MNAht30MHjEgWq9AOtjRCNBwhTgkN3bCkoVIa+AcmsrFJEx5K0xhn2KTSjqlOcfbHCd0iA+1waEuLwyGqX0kD9rn+XgcHjYJqepyDzubDeNunBPDEsrCI4LAzRDUHydLcFwond4ZpoYJBKLJxGvpoM0CzUugt4PE6UEqcD0GTwsJxxo5Z4GUljXDqgAUJO9CBoJ1JM7PjRnFUjPMANA5u5qepk48zaDFwF55Q6uU00DkAJ0F1xqoKjwuDzCoJzcCJApKFcMh01fLlmCZUjgjQaWRZRIKUBE4QEZcmrqiOY7qmEGGeJTz0CB0R0CwhUEP8bttFh7jGYXs/+A9cFNUMSQRHXScSQlNNQywCjgUpTwpM5YuyUtFgpaLBSkWDlYoGKxUNVioarFQ0WKlosFLRYKWiwUpFg5WKBisVDVYqGqxUNFipaLBS0WClosFKRYOVigYrFQ1WKhqsVDRYqWiwUtFgpaLBSkWDlYoGKxWLQ4mESnYS/S+aK/ks/CCUhQAAAABJRU5ErkJggg==" alt="Seattle Proton Center facility photo">
```

Comment:
Images should be placed in a separate img/ directory (AKA folder), not encoded into the html.